File: easypermissions/src/main/java/com/sunplus/easypermissions/AfterPermissionGranted.java
package com.sunplus.easypermissions;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by w.feng on 2018/10/10
 * Email: fengweisb@gmail.com
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.METHOD)
public @interface AfterPermissionGranted {

  int value();
}

File: easypermissions/src/main/java/com/almosting/easypermissions/AfterPermissionGranted.kt
package com.almosting.easypermissions

import kotlin.annotation.AnnotationTarget.FUNCTION
import kotlin.annotation.AnnotationTarget.PROPERTY_GETTER
import kotlin.annotation.AnnotationTarget.PROPERTY_SETTER

/**
 * Created by w.feng on 2018/10/10
 * Email: fengweisb@gmail.com
 */
@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
@Target(
  FUNCTION,
  PROPERTY_GETTER,
  PROPERTY_SETTER
)
annotation class AfterPermissionGranted(val value: Int)